tinygrad/lazy.py

```diff
@@ -143,8 +143,7 @@ def key(self):
 
   def _device_extra_args(self) -> Dict[str, str]: return {"device": self.device.split(":", 1)[1]} if ":" in self.device else {}
 
-  @property
-  def buffers(self) -> Tuple[LazyBuffer, ...]: return (self,)
+  def buffers(self,_=None) -> Tuple[LazyBuffer, ...]: return (self,)
```

Comment:
Why can't this stay a property?

Reply:
I can even put it as a regular property in the __init__ since it always just returns itseld